internal/cmd/auth.go

```diff
@@ -224,9 +224,10 @@ func beginAuth(port int, headless bool, path string) (string, error) {
 			"redirect": {"true"},
 			"type":     {"cli"},
 		}.Encode()
+		browser.Stderr = nil
 		err = browser.OpenURL(authUrl.String())
 		if err != nil {
-			fmt.Println("error: Unable to open browser.")
+			return "", fmt.Errorf("failed to open browser, please use the `--headless` command line option")
```

Comment:
lets print the full command? 

```suggestion
			return "", fmt.Errorf("failed to open browser, please use the `--headless` command line option")
```